python/lsst/obs/ctio0m9/ingest.py

```diff
@@ -58,6 +58,35 @@ def translate_visit(self, md):
         """
         return mjdToVisit(md.get("DATE-OBS"))
 
+    def translate_imgType(self, md):
+        """Get the image type (e.g. bias, dark, flat etc) from the header
+        Translator function derived from a very small dataset from the observatory
+        i.e. may well need adding to when new string values are found
+        """
+        val = md.get("IMAGETYP").rstrip().lstrip()
+        conversion = {'dflat': 'flat', 'DOME FLAT': 'flat',
+                      'zero': 'bias',
+                      'object': 'object'}
+        if val in conversion:
+            return conversion[val]
+        else:
+            return 'NOT_TRANSLATED'
```

Comment:
OK, I've added a warning, for now. It is known that this translator will need to be added to though, as/when new values are found in the metadata.